services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/team/MembershipInvitationManagerImplTest.java

```diff
@@ -261,6 +263,17 @@ public void testGetOpenSubmissionsForTeamAndRequesterInRangeUnauthorized() throw
 	@Test
 	public void testCreateInvitationNotification() throws Exception {
 		MembershipInvtnSubmission mis = createMembershipInvtnSubmission(MIS_ID);
+		testCreateInvitationNotificationHelper(mis);
+	}
+	
+	@Test
+	public void testCreateInvitationNotificationWithNullCreatedOn() throws Exception {
+		MembershipInvtnSubmission mis = createMembershipInvtnSubmission(MIS_ID);
+		mis.setCreatedOn(null);
+		testCreateInvitationNotificationHelper(mis);
```

Comment:
The helper method creates the expected string after calling createInvitationNotification, and createInvitationNotification should always make sure that createdOn is set. That's why the test should always succeed and be stable.